src/pivot-table/components/cells/styles.tsx

```diff
@@ -13,6 +13,9 @@ interface StyledHeaderCellWrapperProps {
   isDimension: boolean;
 }
 
+export const COLUMN_ADJUSTER_CLASS = "sn-pivot-table-column-adjuster";
+const COLUMN_ADJUSTER_BORDER_CLASS = `& .${COLUMN_ADJUSTER_CLASS}-border`;
```

Comment:
export and consume in `ColumnAdjuster.tsx`?

Reply:
I guess the ampersand would have to be solved differently tho

Reply:
yeah exactly, I can try though